docs/Actions/grant-user-role.md

```diff
@@ -0,0 +1,258 @@
+---
+id: grant-user-role
+title: Grant User Role
+sidebar_label: Grant User Role
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `Permission Scheme`
+
+## `Typical Use Cases`
+
+- Assign roles to new users on registration
+- Give access to resources for specific users
+- Temporary give access to users
+
+## `Related Actions`
+
+| Action Name                                             | Description                                                        |
+| ------------------------------------------------------- | ------------------------------------------------------------------ |
+| [Load Users](/docs/Actions/load-user.md)                | Can be used to grant roles to a single user or to a list of users. |
+| [User Registration](/docs/Actions/user-registration.md) | Can be used to grant roles to a newly created user.                |
+
+## `Input Parameter Reference`
+
+| Parameter        | Description                                                                                    | Supports Variables |
+| ---------------- | ---------------------------------------------------------------------------------------------- | ------------------ |
+| Role             | Select the security role, from existing roles, to grant/revoke.                                | Yes                |
+| Other Role Names | Freely input the role name(s) you want to grant/revoke. Multiple roles are separated by comma. | Yes                |
+| Role Validity    | Set a validity period to the granted role.                                                     | No                 |
```

Comment:
How is the validity expressed? 
What happens when it expires?
How would I extend it?

Reply:
I see some of these questions are in the FAQ. Let's move the response here.

Reply:
Updated with the answers through text